fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/utils/JcrPropertyStatementListener.java

```diff
@@ -160,6 +156,14 @@ public void removedStatement(final Statement s) {
 
     }
 
+    private void removeMixin(final FedoraResource resource, final Resource mixinResource, final Resource s) {
```

Comment:
Maybe instead of in a method on this class, the try-catch should be moved back into `JcrRdfTools`?